projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx

```diff
@@ -59,7 +59,7 @@ export const DotFormFields = {
             options={field.values}
             required={field.required}
             required-message={field.requiredMessage}
-            size={+field.size}
+            size={field.size}
```

Comment:
This was crashing the page, not sure why.

Reply:
did some tests an no idea why I set this, anyway this works!